feature-settings-impl/src/main/java/io/novafoundation/nova/feature_settings_impl/presentation/cloudBackup/BackupSettingsViewModel.kt

```diff
@@ -90,7 +110,26 @@ class CloudBackupSettingsViewModel(
     }
 
     fun cloudBackupManageClicked() {
-        TODO()
+        when (syncedState.value) {
+            BackupSyncOutcome.Ok,
```

Comment:
We should show both "change password" and "delete backup" for Ok status as well

Reply:
Yes, I already fixed this in next pr